### ISSUE TITLE
-decay_method noam

### ISSUE THREAD

**Issue by tanaka-jp** [OpenNMT/OpenNMT-py#869]
I tried Transformer without [ -decay_method noam] ,
and the training does not work at all.

Can anyone tell me why decay_method is so important ?

**Comment by vince62s** [comment]
it should work with a regular Adam optimizer / adequate learning rate
what is your command line ?

**Comment by tanaka-jp** [comment]
My command line is here.

python train.py -gpuid 0 -coverage_attn -data data_do_all1all2_baseline2_char_short/all1all2_baseline2_char -encoder_type transformer -decoder_type transformer -train_steps 100000000 -valid_steps 1000 -save_checkpoint_steps 1000 -keep_checkpoint 3 -global_attention mlp -layers 4 -learning_rate 2 -rnn_size 512 -share_embeddings -word_vec_size 512 -dropout 0.1 -optim adam -param_init 0.1 -param_init_glorot -accum_count 4 -normalization tokens -adam_beta2 0.998 -max_generator_batches 2 -position_encoding -warmup_steps 8000 -valid_batch_size 8 -batch_size 16


[2018-08-01 18:29:48,221 INFO] Step 50/100000000; acc:   0.00; ppl: 26881171418161356094253400435962903554686976.00; xent: 6776.20; lr: 2.00000; 4132/2064 tok/s;     52 sec
[2018-08-01 18:29:48,221 INFO] Step 50/100000000; acc:   0.00; ppl: 26881171418161356094253400435962903554686976.00; xent: 6776.20; lr: 2.00000; 4132/2064 tok/s;     52 sec
[2018-08-01 18:30:39,229 INFO] Step 100/100000000; acc:   0.06; ppl: 26881171418161356094253400435962903554686976.00; xent: 6487.71; lr: 2.00000; 4277/1796 tok/s;    103 sec
[2018-08-01 18:30:39,229 INFO] Step 100/100000000; acc:   0.06; ppl: 26881171418161356094253400435962903554686976.00; xent: 6487.71; lr: 2.00000; 4277/1796 tok/s;    103 sec
[2018-08-01 18:31:30,336 INFO] Step 150/100000000; acc:   0.27; ppl: 26881171418161356094253400435962903554686976.00; xent: 5390.18; lr: 2.00000; 4109/1936 tok/s;    154 sec
[2018-08-01 18:31:30,336 INFO] Step 150/100000000; acc:   0.27; ppl: 26881171418161356094253400435962903554686976.00; xent: 5390.18; lr: 2.00000; 4109/1936 tok/s;    154 sec
[2018-08-01 18:32:22,065 INFO] Step 200/100000000; acc:   0.28; ppl: 26881171418161356094253400435962903554686976.00; xent: 4594.07; lr: 2.00000; 4079/1874 tok/s;    206 sec
[2018-08-01 18:32:22,065 INFO] Step 200/100000000; acc:   0.28; ppl: 26881171418161356094253400435962903554686976.00; xent: 4594.07; lr: 2.00000; 4079/1874 tok/s;    206 sec
[2018-08-01 18:33:13,187 INFO] Step 250/100000000; acc:   0.00; ppl: 26881171418161356094253400435962903554686976.00; xent: 5904.60; lr: 2.00000; 4071/1909 tok/s;    257 sec
[2018-08-01 18:33:13,187 INFO] Step 250/100000000; acc:   0.00; ppl: 26881171418161356094253400435962903554686976.00; xent: 5904.60; lr: 2.00000; 4071/1909 tok/s;    257 sec
[2018-08-01 18:34:04,522 INFO] Step 300/100000000; acc:   0.04; ppl: 26881171418161356094253400435962903554686976.00; xent: 7393.47; lr: 2.00000; 4305/2182 tok/s;    308 sec
[2018-08-01 18:34:04,522 INFO] Step 300/100000000; acc:   0.04; ppl: 26881171418161356094253400435962903554686976.00; xent: 7393.47; lr: 2.00000; 4305/2182 tok/s;    308 sec
